azure-pipelines.yml

```diff
@@ -41,15 +41,25 @@ steps:
     dockerHubImageName: $(imageName)
     CC_TEST_REPORTER_ID: $(ccReporterID)
 - script: |
+    set -x
     $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle config --local disable_exec_load true'
     $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake parallel:setup'
-    $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake "parallel:spec[,,--format progress --format RspecJunitFormatter --out rspec-results.xml --no-profile]"'
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake "parallel:spec[,, -O .rspec_parallel_azure]"'
```

Comment:
I feel like we're missing something here. If `rspec` gets an error, it should return an error, but we still need to copy out `rspec-output` to be able to see the errors, right? At the moment we've seen that this build job doesn't fail even if there are errors, so I think what we need to do is:

```
    $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake "parallel:spec[,, -O .rspec_parallel_azure]"'
    test_result=$?
    docker cp s_web_1:/app/rspec-output .
    [ "$test_result" -eq "0" ] && true || false
```

Reply:
Also how is this working in mcfe ... ?

Reply:
Currently it's using a `.rspec_parallel` config file. I'll have a look to see what is currently happening with failing tests.